### DISCUSSION TITLE
Multiple Units in singular ndjson file?

### DISCUSSION THREAD

**Discussion by Jayman391** [synthetichealth/synthea#1324] [Q&A] [answered]
I'm trying to generate an Observation ndjson file with some of the Observation entries having kg and others having lbs in order to implement unit normalization, and I have no idea where to look at this point. Is it in the config file? Please let me know if you've had a similar issue / desire. Thanks!

**Comment by jawalonoski** [comment, marked as answer]
All the observation values and units of measure are controlled by the modules. Except in specific circumstances, for example social determinants of health, observations use the metric system. There are no configuration options to alter this.

Internally, weight is stored as kilograms, and weight is controlled by code, for instance:

https://github.com/synthetichealth/synthea/blob/188ce6373ef9af428975c2f3e9631cac852df41f/src/main/java/org/mitre/synthea/modules/LifecycleModule.java#L498

One option is to export all the observations as kilograms (as per normal), and post process the data to randomly convert some to pounds (using whatever method you want) and use that file as your test data.

**Reply by Jayman391** [reply]
Thanks! Thats what I initially suspected, just wanted to clarify
